src/components/token/utils.ts

```diff
@@ -0,0 +1,55 @@
+import { TokenSymbol } from 'screens/home/TokenImage'
+import { TokenOrBitcoinNetwork } from 'shared/types'
+
+// default order should be RIF, USDRIF, RBTC, BTC and RDOC
+// other tokens should be sorted alphabetically by symbol
+export const sortTokensBySymbol = (
```

Comment:
I dislike this approach because of the readability of the entire file. Can I suggest a different approach? 

We know that there are 4 default tokens (RIF, USDRIF, RBTC, BTC) and then everything else. We know that Rootstock has default tokens listed somewhere, right? (RIF and USDRIF).

So, what about explicitly getting the default tokens and rBTC and BTC and then adding the others. Something very roughly like the below.

```ts
import {defaultTokens} from 'whereever'

const sortTokensBySymbol = (usersTokens) => {
  let list = []

  list.push(defaultTokens)
  list.push(rBTC) // because I don't think it is default
  list.push(BTC) // same reason as above

  // filter, or reduce:
  const otherTokens = usersTokens.filter((token) => {
    token.symbol // where not in existing list
  })

  list.push(otherTokens)

  return list
}
```